docs/using/tutorial/moving-applications.rst

```diff
@@ -4,7 +4,7 @@
 Moving Applications
 ===================
 
-.. note:: If you haven't already, make sure to :doc:`install the flocker-cli package <../installing/index>` before continuing with this tutorial.
+.. note:: If you haven't already, make sure to :ref:`install the flocker-cli package <installflocker>` before continuing with this tutorial.
```

Comment:
This refers to an installation page. It just so happens that the instructions it looks like it should refer to (the CLI instructions) are near the beginning of that page. It should link to the existing `installing-flocker-cli`.